btf_interp/Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2021"
 
 [dependencies]
 btf_types = { path = "../btf_types" }
+tempdir = "0.3.7"
```

Comment:
Does this need to be a `[dependencies]` dep, or could it be a `[dev-dependencies]` dep?  Worth looking up the difference in the Cargo book <https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies>